app/scripts/app.constants.js

```diff
@@ -15,6 +15,7 @@ angular.module('dashboard')
         'PROPOSALSHASUNSAVED': 'PROPOSALSHASUNSAVED',
         'REMARKISUNSAVED': 'REMARKISUNSAVED',
         'PROPOSALDELETED': 'PROPOSALDELETED',
+        'UNSAVEDDATA': 'UNSAVEDDATA',
```

Comment:
Should name be `UNSAVEDPROPOSAL` or similar? On app level it's otherwise hard to know what the feature is about.